projects/core/src/cart/store/effects/cart.effect.ts

```diff
@@ -17,56 +11,127 @@ import { CartActions } from '../actions/index';
 
 @Injectable()
 export class CartEffects {
+  // TODO: remove when removing cart store module
+  @Effect()
+  loadCart2$: Observable<CartActions.LoadMultiCart> = this.actions$.pipe(
+    ofType(CartActions.LOAD_CART),
+    map(
+      (action: CartActions.LoadCart) =>
+        new CartActions.LoadMultiCart(action.payload)
+    )
+  );
+
+  // TODO: change to listen on LoadMultiCart action, remove old actions usage
   @Effect()
   loadCart$: Observable<
     | CartActions.LoadCartFail
+    | CartActions.LoadMultiCartFail
     | CartActions.LoadCartSuccess
+    | CartActions.LoadMultiCartSuccess
     | CartActions.ClearCart
+    | CartActions.ClearMultiCart
+    | CartActions.RemoveCart
   > = this.actions$.pipe(
     ofType(CartActions.LOAD_CART),
-    map(
-      (action: {
-        type: string;
-        payload?: { userId: string; cartId: string };
-      }) => action.payload
-    ),
+    map((action: CartActions.LoadCart) => action.payload),
     mergeMap(payload => {
+      // TODO: get rid of cartData
```

Comment:
Unfortunately, we cannot remove it without breaking change. Let's imagine Customer dipatches action:
```typescript
new LoadCart({cartId: undefined, cartId: undefined})
```
assuming that those values will be filled for him from cartData service

Reply:
I guess cart data service can only be removed for 2.0